lib/fastlane/plugin/wpmreleasetoolkit/actions/common/create_release_action.rb

```diff
@@ -19,7 +19,14 @@ def self.run(params)
           UI.user_error!("Can't find file #{file_path}!") unless File.exist?(file_path)
         end
 
-        Fastlane::Helper::GithubHelper.create_release(repository, version, release_notes, assets, prerelease)
+        Fastlane::Helper::GithubHelper.create_release(
+          repository: repository,
+          version: version,
+          target: params[:target],
```

Comment:
This makes this PR a breaking change, right? Might be a good PR to introduce the `RELEASE_NOTES.txt` file 🙂

Reply:
How so? This is a new param that ends up being optional. So the API is still backwards compatible with any call site not providing `target:`. In fact, we won't have to update the call sites in the app repos at all to benefit from this bugfix.

If anything, this is a change in (1) behavior (before it made those point to `develop`, now to current commit) but that's a bugfix, so in semver this warrants a patch bump (2) API-additive change, not an API-breaking change, since old call sites are still compatible, the new param being optional with default value. So in semver this warrants a minor, non-breaking bump, not a major one anyway 🙂